### DISCUSSION TITLE
A Big Thank You for Your Amazing Library! 🚀💖

### DISCUSSION THREAD

**Discussion by Jenix-Park** [melanchall/drywetmidi#279] [General]
Hey,

I stopped by to check if there's a newer version, but unfortunately, it hasn't arrived yet. 😔 No worries, though—I just wanted to express my gratitude! 💖 You're truly fantastic, and a big thank you for providing this excellent library. 🚀 I've dabbled in creating MIDI features in different languages and crafted a few related plugins, you know the drill. However, I keep returning to DyrWetMIDI because, let's be honest, it's the absolute best! 🎶 Without it, I wouldn't have been able to solve so many problems.

A heartfelt thank you, thank you, thank you! 🙌

**Comment by melanchall** [comment]
Hi,

Thank you for such kind words :-) Unfortunately I can't say when a new version will be released. There are no so much changes there so it will be a minor release anyway.

Thank you,
Max

**Comment by melanchall** [comment]
Hi @Jenix-Park,

**7.0.2** version has been released! 🚀

Thank you,
Max

**Reply by Jenix-Park** [reply]
Wow, thank you!
